taxi/models.py

```diff
@@ -27,6 +30,14 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse("taxi:driver-detail", kwargs={"pk": self.pk})
 
+    def clean_license_number(self):
+        if not re.fullmatch(r"^[A-Z]{3}[0-9]{5}$", self.license_number):
+            raise ValidationError("The license number is incorrect.")
```

Comment:
The `clean_license_number` method correctly validates the `license_number` format. Ensure this method is called within the `clean` method to enforce validation.